### PR TITLE
Delete old code that is not used in the live-results flow

### PR DESCRIPTION
**Pull request by robertbrignull**
This code should all be unused now. The GitHub API method that triggered this code path has also now been removed.

I've done a couple of MRVA runs using this branch and it seems to be working just fine.

### CODE REVIEW

File: .github/workflows/codeql-query.yml [hunk 1]
```diff
@@ -135,7 +135,7 @@ jobs:
           signed_auth_token: ${{ github.event.inputs.signed_auth_token }}
 
       - name: Handle workflow failed
-        if: failure() && github.event.inputs.variant_analysis_id != '' && github.run_attempt == 1
```
Comment (koesie10) [inline .github/workflows/codeql-query.yml:138]: I think we can also make the `variant_analysis_id` required in the `query/action.yml` file, but this can perhaps be done in a different PR since I believe that the `signed_auth_token` should also be required now.

**Review by koesie10 — APPROVED** [review]
LGTM, I also did a MRVA run using this branch and couldn't find any issues with it

File: .github/workflows/codeql-query.yml (hunk 1 above)
Comment (robertbrignull) [reply .github/workflows/codeql-query.yml:138]: Yeah I agree there's some signed auth token changes we can make too. Since we're happy with this as it is I'll merge this now and open a new PR to make both of the fields required.

**Comment by robertbrignull** [comment]
In order to merge this, I removed the branch protection requirement for the `integration-test` CI status that was removed in this PR.

File: .github/workflows/codeql-query.yml (hunk 1 above)
Comment (robertbrignull) [reply .github/workflows/codeql-query.yml:138]: Followup PR: https://github.com/github/codeql-variant-analysis-action/pull/802
